06_deploy_bootstrap_vm.sh

```diff
@@ -28,6 +28,8 @@ done
 IP=$(domain_net_ip ${INFRA_ID}-bootstrap baremetal)
 export API_VIP=$(dig +noall +answer "api.${CLUSTER_DOMAIN}" @$(network_ip baremetal) | awk '{print $NF}')
 echo "address=/api.${CLUSTER_DOMAIN}/${API_VIP}" | sudo tee /etc/NetworkManager/dnsmasq.d/openshift.conf
+export INGRESS_VIP=$(dig +noall +answer "ingress.${CLUSTER_DOMAIN}" @$(network_ip baremetal) | awk '{print $NF}')
+echo "address=/ingress.${CLUSTER_DOMAIN}/${INGRESS_VIP}" | sudo tee -a /etc/NetworkManager/dnsmasq.d/openshift.conf
```

Comment:
```suggestion
echo "address=/apps.${CLUSTER_DOMAIN}/${INGRESS_VIP}" | sudo tee -a /etc/NetworkManager/dnsmasq.d/openshift.conf
```

Reply:
@vrutkovs , I think that we need both the ingress.$CLUSTER_NAME and .apps.$CLUSTER_NAME.
The ingress.$CLUSTER_NAME is used by the keepalive service (to retrieve ingress VIP address).

Reply:
I think it should only be 
```suggestion
echo "address=/.apps.${CLUSTER_DOMAIN}/${INGRESS_VIP}" | sudo tee -a /etc/NetworkManager/dnsmasq.d/openshift.conf
```

WHy do you need the ingress one. .apps is a wildcard entry, so keepalived.sh can resolve it just the same.

Reply:
@celebdor - Done.